app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="text_view_text_size_x_big_35">35sp</dimen>
+    <dimen name="button_text_size_normal">20sp</dimen>
+    <dimen name="text_main_margin_top_normal">30dp</dimen>
+    <dimen name="button_main_margin_top_big">40dp</dimen>
+    <dimen name="button_main_margin_horizontal_xsmall">10dp</dimen>
+</resources>
```

Comment:
to avoid this error you have to add an extra line empty at the end of the file

Reply:
Done